app/src/main/res/layout/activity_talk_topics.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.swiperefreshlayout.widget.SwipeRefreshLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/talk_refresh_view"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/talk_recycler_view"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginStart="16dp"
+            android:layout_marginEnd="16dp"
+            android:scrollbars="vertical" />
+
+        <LinearLayout
+            android:id="@+id/talk_empty_container"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:gravity="center"
+            android:orientation="vertical"
+            android:visibility="gone"
+            tools:visibility="visible">
+
+            <ImageView
+                android:layout_width="120dp"
+                android:layout_height="120dp"
+                android:contentDescription="@null"
+                app:srcCompat="@drawable/ic_user_talk"
+                app:tint="?attr/secondary_text_color"/>
+
+            <TextView
+                style="@style/MaterialMediumSecondaryCaption"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="8dp"
+                android:layout_marginBottom="8dp"
+                android:text="@string/talk_page_empty" />
+
+        </LinearLayout>
```

Comment:
Maybe using a `MaterialTextView` with `app:drawableTopCompat` would be much cleaner?

Reply:
AFAIK it's not possible to change the size of the compound drawables, is it? We need it to be 120dp.

Reply:
It is possible to change its size in the `styles.xml`.  Please see `PageActionTabLayoutButtonStyle` in `styles.xml`. But also, if you think it is not worth to create a `style` for this specific style only, you may leave as it is.